src/geetest.js

```diff
@@ -0,0 +1,75 @@
+// Based on https://github.com/GeeTeam/gt3-server-node-express-bypass/blob/master/app.js
+
+// https://docs.geetest.com/captcha/apirefer/api/server
+// doing this: "If the storage space is not sufficient: Send request to check bypass status before starting the verfication process."
+
+const axios = require("axios")
+const GeetestLib = require("gt3-server-node-express-sdk/sdk/geetest_lib") // galoy fork
+
+async function sendRequest(params) {
+  const request_url = "http://bypass.geetest.com/v1/bypass_status.php"
+  let bypass_res
+  try {
+    const res = await axios({
+      url: request_url,
+      method: "GET",
+      timeout: 5000,
+      params: params,
+    })
+    const resBody = res.status === 200 ? res.data : ""
+    console.log(resBody)
+    bypass_res = resBody["status"]
+  } catch (e) {
+    bypass_res = ""
+  }
+  return bypass_res
+}
+
+class GeeTest {
+  constructor(GEETEST_ID, GEETEST_KEY) {
+    this.GEETEST_ID = GEETEST_ID
+    this.GEETEST_KEY = GEETEST_KEY
+  }
+
+  // private
+  async _getBypassStatus() {
+    const bypass_status = await sendRequest({ gt: this.GEETEST_ID })
+    return bypass_status
+  }
```

Comment:
We are currently using the recommended server implementation that includes a [ByPass mode](https://docs.geetest.com/captcha/deploy/bypass). The [gt4](https://github.com/GeeTeam/gt4_node_express_demo) demo does not include this, and is not well documented yet. And it seems to be incompatible with [gt3](https://docs.geetest.com/captcha/deploy/server/node#Resources), because the backend only makes a [single request](https://github.com/GeeTeam/gt4_node_express_demo/blob/main/routes/index.js#L51) GeeTest.

Reply:
If it isn't compatible with gt3 I think it's a no-go for now since I do not see mobile client sdks for gt4.

Reply:
Something that is left to consider here is if we want to replicate the caching of this value like in the [demo](https://github.com/GeeTeam/gt3-server-node-express-bypass/blob/master/app.js#L12). I don't think is necessary to start, but letting you know.